src/rank_llm/retrieve_and_rerank.py

```diff
@@ -37,7 +37,7 @@ def retrieve_and_rerank(
 
     # Get reranking agent
     reranker = Reranker(
-        Reranker.create_agent(model_path.lower(), default_agent, interactive, **kwargs)
+        Reranker.create_agent(model_path, default_agent, interactive, **kwargs)

```

Comment:
why this change?

Reply:
Reddy's First_Model breaks if it's lower-cased